imports/plugins/included/braintree/lib/collections/schemas/braintree.js

```diff
@@ -43,7 +43,8 @@ export const BraintreePayment = new SimpleSchema({
   },
   cardNumber: {
     type: String,
-    min: 16,
+    min: 12,
+    max: 19,
```

Comment:
Maestro (min 12, max 19) is accepted by PayPal, Auth.net, and Braintree in SOME countries, not all countries. Do we want to us 12/19 to meet this standard where it's accepted, or do we use 13/16 (Visa can be 13 OR 16) and let those using Maestro update it themselves?